packages/contractkit/src/wrappers/Election.ts

```diff
@@ -194,6 +176,7 @@ export class ElectionWrapper extends BaseWrapper<Election> {
       currentVotes.push({
         address: votedGroup,
         votes: voteWeight,
+        capacity: voteWeight,
```

Comment:
why votes and capacity are the same thing?

Reply:
Added a comment